### PR TITLE
8.0 rename to logistics

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: logistic_requisition/wizard/logistic_requisition_cancel.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 from openerp import models, fields, api
 
 
-class LogisticRequisitionCancel(models.TransientModel):
+class LogisticsRequisitionCancel(models.TransientModel):
     """ Ask a reason for the logistic requisition cancellation."""
     _name = 'logistic.requisition.cancel'
```
Comment (lepistone) [inline logistic_requisition/wizard/logistic_requisition_cancel.py:26]: You changed the class name and not the model name, is that intentional?

Comment (yvaucher) [inline logistic_requisition/wizard/logistic_requisition_cancel.py:26]: Ok I will change model names

I'm not sure if we want to change module names too BTW

**Comment by lepistone** [comment]
:+1: once the model names are made consistent with the classes. Thanks

**Comment by jgrandguillaume** [comment]
:+1:

**Pull request merged**